src/topics/async.md

```diff
@@ -0,0 +1,432 @@
+Asynchronous Programming
+========================
+How we perform more than one simultaneous action in Node.js.
+
+Audience: javascript programmers new to node.js.
+
+
+
+What Is Async Programming
+-------------------------
+Asynchronous programming is any technique in which the result of an operation
+is available outside of the normal synchronous (statement by statement) flow
+of code in a program.
```

Comment:
Sounds good. Would you say the problem we're trying to solve is "synchronous programming" or "misunderstanding about asynchronous programming"? (The later is much easier to demonstrate/explain I think :) )